### ISSUE TITLE
Skip already downloaded files

### ISSUE THREAD

**Issue by Jeevhi** [oSumAtrIX/DownOnSpot#18]
Hi, 
it works fine most of the time. However, sometime it stops downloading and it won't download till I restart. However, the it again downloads all previously downloaded songs. 

Is there a way to skip already downloaded files?

**Comment by oSumAtrIX** [comment]
Yeah, pr #14 addresses this issue and adds an option to skip already downloaded files. You can clone his fork and try that, although the fork might not be on the latest version of the main branch. For the time being you can use it though until I merge it.
